test/perf/index.js

```diff
@@ -0,0 +1,34 @@
+/* global window, console */
+/* eslint-disable no-console */
+// hack: prevent example imports from starting their own animation loop
+window.website = true;
+
+const {PerformanceTestRunner} = require('@luma.gl/test-utils');
```

Comment:
This file is a top-level entry point for now, because:
- It only runs in non-headless mode
- The reported fps is easily affected by platform & other running applications. It is not stable enough to be an automatically enforced check along with other tests.

Reply:
Is it a top-level entry point also for Node.js? Does headless-gl support Query timer extension?
* If not then no issue, we can use `import` instead of `require`. 
* If it is, then we could add one more file as a top-level entry point that just `requires` `reify` then this file, to avoid code like `require(...).default`...

Reply:
We need the `require` because of the `window.website = true;` hack. It must be called before any example is imported, otherwise they start their own animation loops right away.

Reply:
The perf test is browser-only at the time because software rendering perf is just not comparable (~1fps in headless Chrome)